crates/apub/src/objects/post.rs

```diff
@@ -197,18 +197,35 @@ impl Object for ApubPost {
       } else {
         None
       };
+
+      let local_site = LocalSite::read(context.pool()).await.ok();
+      let allow_sensitive = local_site_opt_to_sensitive(&local_site);
+      let page_is_sensitive = page.sensitive.unwrap_or(false);
+      let include_image = allow_sensitive || !page_is_sensitive;
```

Comment:
Shouldn't this be `&&` 

Your site should allow sensitive, AND the page should not be sensitive. 

If your site dissallows sensitive, it could still pass the or condition.

Reply:
The intent was to include images when the local site allows sensitive content or the page is not sensitive. Since it short-circuits, this should include both cases:
* Site allows it = images are always included.
* Side doesn't allow it = images are only included if page is not sensitive.

This way non-sensitive images are always proxied by the local site and sensitive images are only proxied when the local site allows it. The idea is to still try to federate some of the load for thumbnails while respecting the local site's configuration. Obviously this depends on the content being flagged appropriately, but that's a larger problem that will require more consideration.

Unless I'm missing something obvious, I believe the logic is correct.

Reply:
Yep looks correct.